docs/mllib-evaluation-metrics.md

```diff
@@ -136,6 +136,7 @@ for all other classes. So, a true positive occurs whenever the prediction and th
 occurs when neither the prediction nor the label take on the value of a given class. By this convention, there can be
 multiple true negatives for a given data sample. The extension of false negatives and false positives from the former
 definitions of positive and negative labels is straightforward.
+Note that the [micro averaged](http://scikit-learn.org/stable/modules/generated/sklearn.metrics.f1_score.html) accuracy, precision, recall, and F1-measure are the same.
```

Comment:
I think I might leave this out. Actually, overall precision/recall/F1 for multiclass are deprecated anyway.